### ISSUE TITLE
SPECs.md needs work

### ISSUE THREAD

**Issue by HughSt** [disarm-platform/fn-village-finder#2]

**Comment by onlyjsmith** [comment]
Still relevant? The file https://github.com/disarm-platform/fn-village-finder/blob/master/SPECS.md seems to contain the basics.

**Comment by HughSt** [comment]
Yep that is about as basic as you can get without being totally useless! Fine for the moment
